documentation/getting-started.md

```diff
@@ -0,0 +1,107 @@
+
+* [1] Done through mutual TLS (mTLS)
+* [2] WoTT does this using an x509 security certificate. These certificates refresh every 7 days.
+
+The certificates are managed entirely by WoTT - the only thing you need to do to begin securing your device is install the WoTT Agent.
+
+## Credentials and the WoTT Dashboard
+
+In an ideal world, we would do away with the need for usernames and passwords entirely and rely on cryptographic identity alone. However, servers cannot verify identity from browsers via cryptographic identity (yet). Servers are unable to verify who you are which is why you are required to login to access for example your email or Google cloud services. The same applies to Web Applications that you as an IoT developer create which can be accessed from any browser.
+
+As a developer, you will likely be accessing a large range of internet/cloud services that require basic HTTP authentication or an API key. In most instances, such credentials are hard coded into applications and can be easily read with access to said applications. WoTT introduces credential management so that you can remotely download the necessary credentials onto your device as a JSON file. This removes the need for hard coded keys and logins from your application. 
+
+We do this via our Dasboard which you can use to monitor your devices. The Dashboard gives your device a trust score (based on how secure it is) as well as giving you the means to add and manage credentials for whatever purpose you need them for.
+
+## Installing the WoTT agent and claiming your device
+
+First, create an account for the [WoTT Dashboard](https://dash.wott.io) and login - you will need this to claim your device later.
+
+Currently, the Agent is only available to Debian package distributions such as Raspbian (for Pi) or Debian. You will also need to have `curl` installed.
+
+To install the agent, run the following in terminal:
+
+```
+$ curl -s https://packagecloud.io/install/repositories/wott/agent/script.deb.sh | sudo bash
+$ sudo apt install wott-agent
+```
+The terminal will now output your WoTT Device ID, claim token, and a link to claim the device. 
+Copy and paste this link into your browser- the fields may already have autofilled. Otherwise fill in the details in the relevant fields.
+
+![claim device](assets docs/claim-device.png)
+
+**Note:** you can view your Device ID and claim token using the following commands as well:
+
+```
+$ wott-agent whoami
+$ wott-agent claim-token
+```
+Your device will now be registered and viewable on the WoTT Dashboard. You can now see the relative trust scores of your devices. To see device specific information, select it and add appropriate tags to these devices (this is relevant for credential management).
+
+See below for an example of a Raspberry Pi with its WoTT meta tags (automatically assigned based on hardware) and the user's personal tags for the Pi's purpose.
+
+![raspberry pi profile](assets docs/rasbpi-profile.png)
```

Comment:
This image reference is wrong. Should be something similar to:

```
{% asset blog/nist.png srcset:width="1300 2x" srcset:width="650 1x" alt="The NIST says so. Prove them wrong if you want" class="img-fluid" %}
```

But you'd have to modify the width for the content.